README.md

```diff
@@ -13,8 +13,9 @@ Only [Active LTS and Current releases][1] are supported.
 # Install
 
 ```
-npm install node-sass gulp-sass --save-dev
+npm install gulp-sass --save-dev
 ```
+You must also install either `node-sass` or `sass`.
```

Comment:
Thoughts on changing this to:

> You can either choose `node-sass` or `sass`:
> ```
> npm install gulp-sass sass --save-dev
> ```
> or:
> ```
> npm install gulp-sass node-sass --save-dev
> ```

This way, users can copy the one-liner they prefer?

Reply:
I think that makes sense. Perhaps we should also add some additional information on the differences?